plugins/Rembed/CMakeLists.txt

```diff
@@ -43,6 +43,7 @@ if(REMBED)
             ./../../system/mp
             ./../../system/jlib)
 
+        set (CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Wno-error=format-nonliteral")
```

Comment:
That's on;y going to fix debug builds - you need to set CMAKE_CXX_FLAGS too I think?